hotel-management/.eslintrc.cjs

```diff
@@ -31,6 +31,7 @@ module.exports = {
   },
   plugins: ['react-refresh', '@typescript-eslint', 'import', 'prettier'],
   rules: {
+    '@typescript-eslint/no-unused-vars': ['error', { varsIgnorePattern: '^React$' }],
```

Comment:
Why we need ignore this pattern `'^React$'`

Reply:
I do not need to import React for the scope of JSX, but the linters might still underline my JSX and tell that 'React' must be in scope, which isn't the case with React 17 I am using. Hence, that's why I have to ignore that pattern.

Reply:
You must disable the rule `React must be in scope` not modify this rule